### ISSUE TITLE
Titanfall 2 Decomplication

### ISSUE THREAD

**Issue by Ma110w** [wfowler1/bsp-decompiler#22]
#15  
>I don't have the game so I can't look. But I do own Titanfall 1, and from what I can see the format is too different. Half of the data is just missing. I don't think they're actually using a binary tree for the levels in these games, so even being just a BSP is misleading.

 Since Titanfall2 is currently the cheapest its ever going to be, why not pick it up right now? the deal ends 19th of December (this month) hell, i could even gift it to you over steam
![image](https://github.com/user-attachments/assets/1cb1f3ce-4991-4587-9337-9f774821cc80)

**Comment by rwynx** [comment]
A couple of devs in Northstar Discord (Titanfall 2 custom server framework) working on maps for some time now, if this issue related to _your_ need for help about this, you can check there first. You can get more detailed information and directions.

**Comment by Ma110w** [comment]
yeah, I'm aware, I dont like really being in that server, nor interacting with it, due to 4z, who seems to like getting people riled up, instead of telling them simply "No."

**Comment by rwynx** [comment]
Hahaha valid crashout not gonna lie, similar reasons for me too, I don't work on/or produce anything related to Titanfall 2 anymore. Well, if you already know where to look I can't be any help then.
